vwifi.c

```diff
@@ -132,14 +180,36 @@ static void owl_scan_routine(struct work_struct *w)
  * This routine is called through workqueue, when the kernel asks to connect
  * through cfg80211_ops.
  */
+static bool is_valid_ssid(const char *connecting_ssid)
+{
+    bool is_valid = false;
+
+    const char *org_ssid_list = (ssid_list != NULL && strlen(ssid_list))
```

Comment:
Ditto.

Reply:
fixed in https://github.com/sysprog21/vwifi/pull/4/commits/a02d3d9611e41eb07f7b7a401bffd387324ad764